java-client/src/main/java/org/scassandra/http/client/PrimingRequest.java

```diff
@@ -66,7 +66,6 @@ public PrimingRequestBuilder withRows(List<Map<String, ?>> rows) {
             return this;
         }
 
-        @SafeVarargs
```

Comment:
SafeVarargs and Objects.hash both require Java7.  Replaced objects.hash with autogenerated equals/hashCode methods in IntelliJ